CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+Thank you for contributing to the Donkeycar project.  Here are some guidelines that can help you be successful in getting your contributions merged into Donkeycar.
+
+**What makes for a good Pull Request?**
+- The code implements a bug fix or optimization/refactor to a preexisting feature or adds a new feature.
+- If the code is adding a new feature, it would be advisable to open an Issue in the Donkeycar repo prior to opening the PR.  This would allow discussion on how the feature may be best implemented and even if the feature is something that would get accepted into Donkeycar.  The new feature should have fairly broad applicability; it should be a feature that would be useful to a lot of users.
```

Comment:
- The code should be well written, and where it is not obvious what is going on and why, have some comments in it. It should also be pep-8 compliant in style. Like with any other code, it should avoid code duplications from itself or other existing code. It should keep data and methods arranged in classes as much as possible and should avoid lengthy, monolithic functions and functions with too many input/output parameters. Monkey-patching is a no-go and pythonic style usually is a good guide.